src/monaco-editor/Editor.tsx

```diff
@@ -19,7 +19,9 @@ const BfEditor = (props: {
 
         const newEditor = monaco.editor.create(editorRef.current!, {
           value:
-            '++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.',
+            'Hello World\n' +
+            '++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.' +
```

Comment:
```suggestion
            '++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++' +
```
The `.` in the URL prints this char